setup.py

```diff
@@ -134,38 +163,36 @@ def _parse_version(filename):
 setup(
     name="ale-py",
     version=_parse_version("CMakeLists.txt"),
-    description="Arcade Learning Environment Python Interface",
+    description="The Arcade Learning Environment (ALE) - a platform for AI research.",
     long_description=_read("README.md"),
     long_description_content_type="text/markdown",
     keywords=["reinforcement-learning", "arcade-learning-environment", "atari"],
     url="https://github.com/mgbellemare/Arcade-Learning-Environment",
     author="Arcade Learning Environment Authors",
-    license="GPL",
-    ext_modules=[
-        CMakeExtension(
-            "ale_py.libale_c",
-            ".",
-            [
-                "-DUSE_SDL=OFF",
-                "-DUSE_RLGLUE=OFF",
-                "-DBUILD_EXAMPLES=OFF",
-                "-DBUILD_CPP_LIB=OFF",
-                "-DBUILD_CLI=OFF",
-                "-DBUILD_C_LIB=ON",
-            ],
-        )
-    ],
-    cmdclass={"build_ext": CMakeBuild},
-    packages=["ale_py"],
-    install_requires=["numpy"],
```

Comment:
Ubuntu 18.04 only installs version 3.10 of cmake. Since currently ale requires version 3.14, perhaps cmake should be made an install requirement?

Reply:
Yeah, the issue is those are runtime dependencies and Python doesn't have a great way to specify build-time dependencies. I'll look into creating a `pyproject.toml` file to specify `cmake` as a build-time dependency.

Reply:
This should be fixed now if you build with pep517. e.g.,
```sh
pip install pep517
python -m pep517.build .
```
This should install the build requirements which includes cmake.